### ISSUE TITLE
Quora.com login

### ISSUE THREAD

**Issue by rpodric** [nextgens/anti-paywall#17]
Looks to be impossible to login into (Chrome, Firefox) with this plugin enabled. If you try, essentially nothing happens. Eventually you'll get into a situation where the login form doesn't even display anymore, but clearing quora cookies fixes that, though you still can't login.

I could be wrong, but I don't think Quora.com even has a login requirement anymore to view posts (just if you want to post and do other things with your account). Like:
https://www.quora.com/What-will-happen-when-China’s-GDP-surpasses-the-US/answer/Godfree-Roberts

It used to be, at least several months ago, that unless you had a special link meant for sharing, you did have to login. So maybe there's no need for the site to be included in this extension now?

**Comment by nextgens** [comment]
I guess it's like #3 ; you are either logged-in or bypassing the paywall... you can't do both.

I am planning to add a UI to allow users to manage the scope/list of sites affected by the extension... It might not happen until next year now though :)

**Comment by rpodric** [comment]
Yes, that makes sense. Funny you mention that UI...I very nearly brought that up. It would be great.

But in the case of Quora, I don't think it has any kind of paywall in the first place. AFAIK, anyone can read stuff there now any which way, so I'm not sure what's being bypassed.

**Comment by Ascar699** [comment]
I can confirm that Antipaywall renders Quora non-accessible.  Just found out by cherry-picking the add-ons in my Waterfox profile trying to find the one disabling login into Quora.  I feel lucky that Antipaywall was on the top of the list as the name starts with "A".
